simplq/src/components/Layout/Routes.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Route, Switch, BrowserRouter as Router } from 'react-router-dom';
-import JoinQueueWithDetails from 'components/pages/Join';
+import JoinQueueWithDetails from 'components/pages/Join/JoinPage';
```

Comment:
Shall we create and use a index.js?

Reply:
Yes. And should we rename folder into JoinPahe?

Reply:
Upto you, it's already in the pages subfolder, it it might be fine even without it. `pages/Join`